### PR TITLE
Add tutorial for SI and SB provisioning through Provisioning API

### PR DESCRIPTION
**Pull request by szwedm**
**Description**

Changes proposed in this pull request:

Add tutorial which shows the process of provisioning service instance and service binding through Provisioning API.

**Related issue(s)**

See #1205

### CODE REVIEW

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+   export UAA_URL=$(btp --format json get services/binding --name cis-local-binding | jq -r '.credentials.uaa.url')
+   export PROVISIONING_SERVICE_URL=$(btp --format json get services/binding --name cis-local-binding | jq -r '.credentials.endpoints.provisioning_service_url')
+   ```
+
+3. Get the access token for Provisioning Service API using the client credentials:
+   ```bash
+   TOKEN=$(curl -s -X POST "${UAA_URL}/oauth/token" -H "Content-Type: application/x-www-form-urlencoded" -u "${CLIENT_ID}:${CLIENT_SECRET}" --data-urlencode "grant_type=client_credentials" | jq -r '.access_token')
+   ```
+
+4. Check if Kyma runtime is available for provisioning:
```
Comment (PK85) [inline docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md:44]: provide payload in this point, all pf the parameters are const

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+7. After the provisioning is completed, create a binding to get the kubeconfig:
+   ```bash
+   curl -s -X PUT "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings" -H "accept: application/json" -H "Authorization: bearer $TOKEN" -H "Content-Type: application/json" -d "{\"serviceInstanceId\":\"$INSTANCE_ID\",\"planId\":\"$PLAN_ID\"}" | jq -r '.credentials.kubeconfig' > kubeconfig.yaml
+   ```
+
+8. Set `KUBECONFIG` environment variable to the path of the kubeconfig file to access the cluster through **kubectl**:
+   ```bash
+   export KUBECONFIG=kubeconfig.yaml
+   ```
```
Comment (PK85) [inline docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md:74]: Add point 9 with example call to get pods in the default namespace. Will be no resources found.

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      ```
+   2. Provision SAP Cloud Management Service instance with Client Credentials grant type passed as parameters: 
+      ```bash
+      btp create services/instance --offering-name cis --plan-name local --name ${CIS_INSTANCE_NAME} --parameters {\"grantType\":\"clientCredentials\"}
+      ```
+   3. Create a binding for the instance:
+      ```bash
+      btp create services/binding --name ${CIS_INSTANCE_NAME}-binding --instance-name ${CIS_INSTANCE_NAME}
+      ```
+
```
Comment (IwonaLanger) [inline docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md:30]:
```suggestion
    > [!NOTE]
    > For alternative methods, see [Getting an Access Token for SAP Cloud Management Service APIs](https://help.sap.com/docs/btp/sap-business-technology-platform/getting-access-token-for-sap-cloud-management-service-apis?&version=Cloud).

```

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md (hunk 2 above)
Comment (IwonaLanger) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md:74]: Shouldn't these be ## Next Steps? Point 10 isn't what you're trying to achieve through this procedure, correct? Not sure about point 9.

Comment (PK85) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md:74]: We want to achieve to show, that invalidating credenrials works as expected

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+# Provision SAP BTP, Kyma Runtime Using SAP Cloud Management Service's Provisioning API
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md:1]: I will change the title to `Manage SAP BTP, Kyma Runtime Using the Provisioning API of the SAP Cloud Management Service` and add instructions how to deprovision the instance and bindings.

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+SAP Cloud Management Service (cis) provides Provisioning Service API to create and manage available environments. This document describes how to use the Provisioning Service API to create and access SAP BTP, Kyma runtime on AWS.
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md:3]: correct, Provisioning Service is a microservice and has its own API.

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md (hunk 1 above)
Comment (szwedm) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md:47]: I added environment variables with const values in the bash command.

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+## Prerequisites
+Your subaccount must have [entitlements](https://help.sap.com/docs/btp/sap-business-technology-platform/managing-entitlements-and-quotas-using-cockpit) for [SAP BTP, Kyma runtime](https://discovery-center.cloud.sap/index.html#/serviceCatalog/kyma-runtime) and [SAP Cloud Management Service for SAP BTP
+](https://discovery-center.cloud.sap/index.html#/serviceCatalog/cloud-management-service).
+
+### CLI tools
+- [jq](https://jqlang.github.io/jq/)
+- [curl](https://curl.se/)
+- [btp CLI](https://help.sap.com/docs/btp/sap-business-technology-platform/download-and-start-using-btp-cli-client?locale=en-US) (optional)
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+## Steps
+
+1. Provision SAP Cloud Management Service instance with `local` plan and create a binding to get the credentials for Provisioning Service API. The procedure is described in [Help Portal](https://help.sap.com/docs/btp/sap-business-technology-platform/getting-access-token-for-sap-cloud-management-service-apis) or you can use btp CLI as shown below:
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+   1. Set `CIS_INSTANCE_NAME` environment variable with the name of the SAP Cloud Management Service instance:
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+      ```bash
+      export CIS_INSTANCE_NAME={CIS_INSTANCE_NAME}
+      ```
+   2. Provision SAP Cloud Management Service instance with Client Credentials grant type passed as parameters:
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+   2. Provision SAP Cloud Management Service instance with Client Credentials grant type passed as parameters: 
+      ```bash
+      btp create services/instance --offering-name cis --plan-name local --name ${CIS_INSTANCE_NAME} --parameters {\"grantType\":\"clientCredentials\"}
+      ```
+   3. Create a binding for the instance:
+      ```bash
+      btp create services/binding --name ${CIS_INSTANCE_NAME}-binding --instance-name ${CIS_INSTANCE_NAME}
+      ```
+
+2. Set `CLIENT_ID`, `CLIENT_SECRET`, `UAA_URL`, `PROVISIONING_SERVICE_URL` environment variables using the credentials from the binding stored in `clientid`, `clientsecret`, `url`, `provisioning_service_url` fields. You can use btp CLI to get the credentials as shown below:
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+   ```bash
+   INSTANCE_ID=$(curl -s -X POST "$PROVISIONING_SERVICE_URL/provisioning/v1/environments" -H "accept: application/json" -H "Authorization: bearer $TOKEN" -H "Content-Type: application/json" -d "{\"environmentType\":\"kyma\",\"parameters\":{\"name\":\"$NAME\",\"region\":\"$REGION\"},\"planName\":\"$PLAN\",\"serviceName\":\"kymaruntime\",\"user\":\"$USER_ID\"}" | jq -r '.id')
+   ```
+
+7. After the provisioning is completed, create a binding to get the kubeconfig:
+   ```bash
+   curl -s -X PUT "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings" -H "accept: application/json" -H "Authorization: bearer $TOKEN" -H "Content-Type: application/json" -d "{\"serviceInstanceId\":\"$INSTANCE_ID\",\"planId\":\"$PLAN_ID\"}" | jq -r '.credentials.kubeconfig' > kubeconfig.yaml
+   ```
+
+8. Set `KUBECONFIG` environment variable to the path of the kubeconfig file to access the cluster through **kubectl**:
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+
+2. Set `CLIENT_ID`, `CLIENT_SECRET`, `UAA_URL`, `PROVISIONING_SERVICE_URL` environment variables using the credentials from the binding stored in `clientid`, `clientsecret`, `url`, `provisioning_service_url` fields. You can use btp CLI to get the credentials as shown below:
+   ```bash
+   export CLIENT_ID=$(btp --format json get services/binding --name cis-local-binding | jq -r '.credentials.uaa.clientid')
+   export CLIENT_SECRET=$(btp --format json get services/binding --name cis-local-binding | jq -r '.credentials.uaa.clientsecret')
+   export UAA_URL=$(btp --format json get services/binding --name cis-local-binding | jq -r '.credentials.uaa.url')
+   export PROVISIONING_SERVICE_URL=$(btp --format json get services/binding --name cis-local-binding | jq -r '.credentials.endpoints.provisioning_service_url')
+   ```
+
+3. Get the access token for Provisioning Service API using the client credentials:
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md:82]: suggestion applied.

File: docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md (hunk 3 above)
Comment (szwedm) [reply docs/user/05-70-provisioning-kyma-environment-through-cis-provisioning-service.md:30]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 13]
```diff
@@ -0,0 +1,120 @@
...
+      ```bash
+      btp create services/instance --offering-name cis --plan-name local --name ${CIS_INSTANCE_NAME} --parameters {\"grantType\":\"clientCredentials\"}
+      ```
+   3. Create a binding for the instance:
+      ```bash
+      btp create services/binding --name ${CIS_INSTANCE_NAME}-binding --instance-name ${CIS_INSTANCE_NAME}
+      ```
+
+      > [!NOTE]
+      > For alternative methods, see [Getting an Access Token for SAP Cloud Management Service APIs](https://help.sap.com/docs/btp/sap-business-technology-platform/getting-access-token-for-sap-cloud-management-service-apis?&version=Cloud).
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: fixed.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 14]
```diff
@@ -0,0 +1,120 @@
...
+
+2. Set **CLIENT_ID**, **CLIENT_SECRET**, **UAA_URL**, **PROVISIONING_SERVICE_URL** environment variables using the credentials from the binding stored in **clientid**, **clientsecret**, **url**, **provisioning_service_url** fields. Use the btp CLI to get the credentials:
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 15]
```diff
@@ -0,0 +1,120 @@
...
+The SAP Cloud Management service (technical name: `cis`) provides the Provisioning Service API to create and manage available environments. Use the Provisioning Service API to manage and access SAP BTP, Kyma runtime on AWS.
+
+## Prerequisites
+- Your subaccount must have entitlements for SAP BTP, Kyma runtime and SAP Cloud Management Service for SAP BTP. See [Managing Entitlements and Quotas Using the Cockpit](https://help.sap.com/docs/btp/sap-business-technology-platform/managing-entitlements-and-quotas-using-cockpit?&version=Cloud).
+
+- CLI tools
+  - [kubectl](https://kubernetes.io/docs/reference/kubectl/)
+  - [jq](https://jqlang.github.io/jq/)
+  - [curl](https://curl.se/)
+  - [btp CLI](https://help.sap.com/docs/btp/sap-business-technology-platform/download-and-start-using-btp-cli-client?locale=en-US) (optional)
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 16]
```diff
@@ -0,0 +1,120 @@
...
+
+## Steps
+
+1. Use the btp CLI to provision the SAP Cloud Management service instance with the `local` plan and create a binding to get the credentials for the Provisioning Service API:
+
+   1. Set **CIS_INSTANCE_NAME** environment variable with the name of the SAP Cloud Management service instance:
+      ```bash
+      export CIS_INSTANCE_NAME={CIS_INSTANCE_NAME}
+      ```
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 17]
```diff
@@ -0,0 +1,120 @@
...
+   2. Provision the SAP Cloud Management service instance with the Client Credentials grant type passed as parameters: 
+      ```bash
+      btp create services/instance --offering-name cis --plan-name local --name ${CIS_INSTANCE_NAME} --parameters {\"grantType\":\"clientCredentials\"}
+      ```
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 18]
```diff
@@ -0,0 +1,120 @@
...
+   3. Create a binding for the instance:
+      ```bash
+      btp create services/binding --name ${CIS_INSTANCE_NAME}-binding --instance-name ${CIS_INSTANCE_NAME}
+      ```
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 19]
```diff
@@ -0,0 +1,120 @@
...
+
+3. Get the access token for the Provisioning Service API using the client credentials:
+   ```bash
+   TOKEN=$(curl -s -X POST "${UAA_URL}/oauth/token" -H "Content-Type: application/x-www-form-urlencoded" -u "${CLIENT_ID}:${CLIENT_SECRET}" --data-urlencode "grant_type=client_credentials" | jq -r '.access_token')
+   ```
+
+4. Check if Kyma runtime is available for provisioning:
+   ```bash
+   curl -s "$PROVISIONING_SERVICE_URL/provisioning/v1/availableEnvironments" -H "accept: application/json" -H "Authorization: bearer $TOKEN" | jq
+   ```
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 20]
```diff
@@ -0,0 +1,120 @@
...
+
+5. Set **ENVIRONMENT_TYPE**, **SERVICE_NAME** environment variables to const values **kyma**, **kymaruntime** and provide values for **NAME**, **REGION**, **PLAN**, **USER_ID** environment variables:
+   ```bash
+   export ENVIRONMENT_TYPE="kyma"
+   export SERVICE_NAME="kymaruntime"
+   export NAME={RUNTIME_NAME}
+   export REGION={CLUSTER_REGION}
+   export PLAN={KYMA_RUNTIME_PLAN_NAME}
+   export USER_ID={USER_ID}
+   ```
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 21]
```diff
@@ -0,0 +1,120 @@
...
+
+6. Provision the Kyma runtime and save the instance ID in **INSTANCE_ID** environment variable:
+   ```bash
+   INSTANCE_ID=$(curl -s -X POST "$PROVISIONING_SERVICE_URL/provisioning/v1/environments" -H "accept: application/json" -H "Authorization: bearer $TOKEN" -H "Content-Type: application/json" -d "{\"environmentType\":\"$ENVIRONMENT_TYPE\",\"parameters\":{\"name\":\"$NAME\",\"region\":\"$REGION\"},\"planName\":\"$PLAN\",\"serviceName\":\"$SERVICE_NAME\",\"user\":\"$USER_ID\"}" | jq -r '.id')
+   ```
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 22]
```diff
@@ -0,0 +1,120 @@
...
+
+7. Optionally, set **EXPIRATION_SECONDS** environment variable to the number of seconds (max 7200) after which a binding will expire:
+   ```bash
+   export EXPIRATION_SECONDS={EXPIRATION_SECONDS}
+   ```
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 23]
```diff
@@ -0,0 +1,120 @@
...
+
+8. After the provisioning is completed, create the binding to get the kubeconfig:
+   ```bash
+   [ -z "$EXPIRATION_SECONDS" ] && \
+   curl -s -X PUT "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings" -H "accept: application/json" -H "Authorization: bearer $TOKEN" | jq -r '.credentials.kubeconfig' > kubeconfig.yaml || \
+   curl -s -X PUT "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings" -H "accept: application/json" -H "Authorization: bearer $TOKEN" -H "Content-Type: application/json" -d "{\"parameters\":{\"expiration_seconds\":\"$EXPIRATION_SECONDS\"}}" | jq -r '.credentials.kubeconfig' > kubeconfig.yaml
+   ```
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 24]
```diff
@@ -0,0 +1,120 @@
...
+
+9. To access the cluster through kubectl, set the **KUBECONFIG** environment variable to the path of the kubeconfig file:
+   ```bash
+   export KUBECONFIG=kubeconfig.yaml
+   ```
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 25]
```diff
@@ -0,0 +1,120 @@
...
+
+10. Verify the connection to the cluster by running a kubectl command to get pods:
+   ```bash
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 26]
```diff
@@ -0,0 +1,120 @@
...
+   kubectl get pods
+   ```
+
+    kubectl should return the list of pods in default namespace running in the cluster meaning that the cluster is accessible.
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md:103]: right, it should not be a code block, corrected.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 27]
```diff
@@ -0,0 +1,120 @@
...
+
+   > [!NOTE]
+   > Following steps are optional and show how to revoke the credentials by deleting the binding.
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: fixed.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 28]
```diff
@@ -0,0 +1,120 @@
...
+
+11. Get the ID of the binding from the instance's bindings list and save it in **BINDING_ID** environment variable:
+   ```bash
+   curl -s "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings" -H "accept: application/json" -H "Authorization: bearer $TOKEN"
+   ```
+   ```bash
+   export BINDING_ID={BINDING_ID}
+   ```
+
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 29]
```diff
@@ -0,0 +1,120 @@
...
+12. Delete the binding to revoke the credentials:
+   ```bash
+   curl -s -X DELETE "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings/$BINDING_ID" -H "accept: application/json" -H "Authorization: bearer $TOKEN"
+   ```
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md (hunk 26 above)
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 30]
```diff
@@ -0,0 +1,120 @@
...
+
+    Try to access the cluster using kubectl. The connection should be refused, meaning that the binding was successfully deleted and credentials revoked:
+   ```bash
+   kubectl get pods
+   ```
+
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: correct, should not be a code block. Fixed.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 31]
```diff
@@ -0,0 +1,120 @@
...
+## Next Steps
+
+Delete bindings for the instance as shown in steps 11 and 12. To deprovision the Kyma runtime, use the following command:
+   ```bash
+   curl -s -X DELETE "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID" -H "accept: application/json" -H "Authorization: bearer $TOKEN"
+   ```
```
[the remark this answers is not in the document]
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: suggestion applied.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 32]
```diff
@@ -0,0 +1,139 @@
...
+   curl -s -X PUT "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings" -H "accept: application/json" -H "Authorization: bearer $TOKEN" -H "Content-Type: application/json" -d "{\"parameters\":{\"expiration_seconds\":$EXPIRATION_SECONDS}}" | jq -r '.credentials.kubeconfig' > kubeconfig.yaml
+   ```
+
+9. To access the cluster through kubectl, set the **KUBECONFIG** environment variable to the path of the kubeconfig file:
+
+   ```bash
+   export KUBECONFIG=kubeconfig.yaml
+   ```
+
+10. Verify the connection to the cluster by running a kubectl command to get pods:
```
Comment (IwonaLanger) [inline docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md:97]: Starting from point 10, the code blocks are not indented properly. I think you need to add one more space at the beginning of these lines.
Screenshot 2024-11-05 at 16 00 37

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 33]
```diff
@@ -0,0 +1,139 @@
...
+   export BINDING_ID={BINDING_ID}
+   ```
+
+12. Delete the binding to revoke the credentials:
+
+   ```bash
+   curl -s -X DELETE "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings/$BINDING_ID" -H "accept: application/json" -H "Authorization: bearer $TOKEN"
+   ```
+
+   Try to access the cluster using kubectl. The connection should be refused, which means that the binding was successfully deleted and credentials revoked:
```
Comment (IwonaLanger) [inline docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md:124]: Not indented properly.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md [hunk 34]
```diff
@@ -0,0 +1,139 @@
...
+   export KUBECONFIG=kubeconfig.yaml
+   ```
+
+10. Verify the connection to the cluster by running a kubectl command to get pods:
+
+   ```bash
+   kubectl get pods
+   ```
+
+   kubectl should return the list of Pods in the `default` namespace running in the cluster, ehich means that the cluster is accessible.
```
Comment (IwonaLanger) [inline docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md:103]: Not indented properly.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md (hunk 32 above)
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md:99]: corrected.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md (hunk 33 above)
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: corrected.

File: docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md (hunk 34 above)
Comment (szwedm) [reply docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md]: corrected.

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**